src/Sentry/Platforms/Cocoa/CocoaProfiler.cs

```diff
@@ -0,0 +1,62 @@
+using Sentry.Extensibility;
+using Sentry.Internal;
+using Sentry.Cocoa.Extensions;
+using Sentry.Cocoa.Facades;
+
+namespace Sentry.Cocoa;
+
+internal class CocoaProfiler : ITransactionProfiler
+{
+    private readonly SentryOptions _options;
+    private readonly SentryId _traceId;
+    private readonly CocoaSdk.SentryId _cocoaTraceId;
+    private readonly ulong _startTimeNs;
+    private ulong _endTimeNs;
+    private readonly SentryStopwatch _stopwatch = SentryStopwatch.StartNew();
```

Comment:
just FYI this will be the last thing to run in the contructor (iirc). I'm pretty sure it doesn't matter but if u wanted this to be the first to include time spent in the ctor (logger is the only thing that would matter), add to it explicitlty

Reply:
thx, I'll change that